### ISSUE TITLE
Siesta/4.1-b4-foss-2018b

### ISSUE THREAD

**Issue by JJImenezna** [easybuilders/easybuild#727]
Hello!

Installin this module appears the error:

ERROR: Build of /opt/shared/apps/EasyBuild/4.4.1/easybuild/easyconfigs/s/Siesta/Siesta-4.1-b4-foss-2018b.eb failed (err: "build failed (first 300 chars): Failed to copy file [path]/Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/Util/STM/ol-stm/Src/stm to Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/bin: [Errno 2] No such file or directory: '[path]/Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/Util/STM/")

Any help will be really appreciated
Best Regards

**Comment by ocaisa** [comment]
Does the directory `[path]/Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/Util` contain anything? Are there errors earlier in the log?

**Comment by ocaisa** [comment]
If the file exists on disk, are you perhaps using an NFS directory as a build path? This can sometimes lead to these kinds of problems.

**Comment by JJImenezna** [comment]
> 
> 
> Does the directory `[path]/Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/Util` contain anything? Are there errors earlier in the log?

Yes. there are several folders wtihin. I do not see any error regarding this route.

Many  thanks 4 your responde

**Comment by JJImenezna** [comment]
> 
> 
> If the file exists on disk, are you perhaps using an NFS directory as a build path? This can sometimes lead to these kinds of problems.

I am using NFS exports. However the log references to /Util/STM/ol-stm/Src/stm 
and the existing files are:
/Util/STM/ol-stm/Src/stm.o
/Util/STM/ol-stm/Src/stm.f

Many thanks for your response

**Comment by ocaisa** [comment]
The error reported was
```
No such file or directory: '[path]/Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/Util/STM/"
```
and according to what you said this directory does actually exist, I would then put this down to NFS. I suggest you add the option `--buildpath=/tmp` to your `eb` command and try again.

**Comment by JJImenezna** [comment]
> 
> 
> The error reported was
> 
> ```
> No such file or directory: '[path]/Siesta/4.1-b4/foss-2018b/siesta-4.1-b4/Util/STM/"
> ```
> 
> and according to what you said this directory does actually exist, I would then put this down to NFS. I suggest you add the option `--buildpath=/tmp` to your `eb` command and try again.

I am trying with --buildpath and --rebuild but it is not working. 
I will tray again and let you know
Many thanks
